src/components/global/search/search.hbs

```diff
@@ -1,5 +1,8 @@
-<div class="search">
-	<h2 class="visuallyhidden">Suche</h2>
+<div class="search js-search">
```

Comment:
We should add a `role="search"` here (which is one of the very few ARIA landmarks that don't have an HTML5 equivalent).